CONTRIBUTORS.md

```diff
@@ -8,15 +8,16 @@
  - [flemse](https://github.com/flemse)
  - [bfayers](https://github.com/bfayers)
  - [Bond_009](https://github.com/Bond-009)
+ - [AnthonyLavado](https://github.com/anthonylavado)
 
 # Emby Contributors
 
- - [LukePulverenti](https://github.com/LukePulverenti) 
```

Comment:
I'd rather we don't touch their lines, so that `git blame` works correctly.

Reply:
Ok - I see the issue now. Will change when I get a chance to.